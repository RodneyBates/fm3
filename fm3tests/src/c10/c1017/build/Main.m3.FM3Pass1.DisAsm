 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L1(16_1),[1,0])
 ItkModuleBeginPatch(C743,L1(16_1),[20,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkOpenScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[21,2])
 ItkStmtListLt(L0(16_0),[21,2])
 ItkOpenDeclListLtPatch(C10,L14(16_e),[5,2])
 ItkOpenDeclListRt(L14(16_e),[5,2])
 ItkConstDeclLtPatch(C688,[18,8])
 ItkConstDeclRt([18,8])
 ItkLiteral(104 StkWideTextLit,(16_1),[18,12])
 ItkConstDeclValue([18,10])
 ItkDeclTypeAbsent([18,10])
 ItkConstDeclType([18,10])
 ItkDeclId(L9(16_9),Id15"N",L-9223372036854775808(16_8000000000000000),[18,8])
 ItkOpenDeclListSep(L13(16_d),[5,2])
 ItkConstDeclLtPatch(C639,[17,8])
 ItkConstDeclRt([17,8])
 ItkLiteral(103 StkTextLit,(16_1),[17,12])
 ItkConstDeclValue([17,10])
 ItkDeclTypeAbsent([17,10])
 ItkConstDeclType([17,10])
 ItkDeclId(L9(16_9),Id14"M",L-9223372036854775808(16_8000000000000000),[17,8])
 ItkOpenDeclListSep(L12(16_c),[5,2])
 ItkConstDeclLtPatch(C587,[16,8])
 ItkConstDeclRt([16,8])
 ItkLiteral(106 StkWideCharLit,(16_10ffff),[16,12])
 ItkConstDeclValue([16,10])
 ItkDeclTypeAbsent([16,10])
 ItkConstDeclType([16,10])
 ItkDeclId(L9(16_9),Id13"L",L-9223372036854775808(16_8000000000000000),[16,8])
 ItkOpenDeclListSep(L11(16_b),[5,2])
 ItkConstDeclLtPatch(C536,[15,8])
 ItkConstDeclRt([15,8])
 ItkLiteral(106 StkWideCharLit,(16_ffff),[15,12])
 ItkConstDeclValue([15,10])
 ItkDeclTypeAbsent([15,10])
 ItkConstDeclType([15,10])
 ItkDeclId(L9(16_9),Id12"K",L-9223372036854775808(16_8000000000000000),[15,8])
 ItkOpenDeclListSep(L10(16_a),[5,2])
 ItkConstDeclLtPatch(C487,[14,8])
 ItkConstDeclRt([14,8])
 ItkLiteral(105 StkCharLit,(16_0),[14,12])
 ItkConstDeclValue([14,10])
 ItkDeclTypeAbsent([14,10])
 ItkConstDeclType([14,10])
 ItkDeclId(L9(16_9),Id11"J",L-9223372036854775808(16_8000000000000000),[14,8])
 ItkOpenDeclListSep(L9(16_9),[5,2])
 ItkConstDeclLtPatch(C438,[13,8])
 ItkConstDeclRt([13,8])
 ItkLiteral(105 StkCharLit,(16_24),[13,12])
 ItkConstDeclValue([13,10])
 ItkDeclTypeAbsent([13,10])
 ItkConstDeclType([13,10])
 ItkDeclId(L9(16_9),Id10"I",L-9223372036854775808(16_8000000000000000),[13,8])
 ItkOpenDeclListSep(L8(16_8),[5,2])
 ItkConstDeclLtPatch(C381,[12,8])
 ItkConstDeclRt([12,8])
 ItkLiteral(102 StkExtendedLit,(16_412ac0149999999a),[12,12])
 ItkConstDeclValue([12,10])
 ItkDeclTypeAbsent([12,10])
 ItkConstDeclType([12,10])
 ItkDeclId(L9(16_9),Id9"H",L-9223372036854775808(16_8000000000000000),[12,8])
 ItkOpenDeclListSep(L7(16_7),[5,2])
 ItkConstDeclLtPatch(C324,[11,8])
 ItkConstDeclRt([11,8])
 ItkLiteral(101 StkLongRealLit,(16_405edccccccccccd),[11,12])
 ItkConstDeclValue([11,10])
 ItkDeclTypeAbsent([11,10])
 ItkConstDeclType([11,10])
 ItkDeclId(L9(16_9),Id8"G",L-9223372036854775808(16_8000000000000000),[11,8])
 ItkOpenDeclListSep(L6(16_6),[5,2])
 ItkConstDeclLtPatch(C271,[10,8])
 ItkConstDeclRt([10,8])
 ItkLiteral(100 StkRealLit,(16_3dcccccd),[10,12])
 ItkConstDeclValue([10,10])
 ItkDeclTypeAbsent([10,10])
 ItkConstDeclType([10,10])
 ItkDeclId(L9(16_9),Id7"F",L-9223372036854775808(16_8000000000000000),[10,8])
 ItkOpenDeclListSep(L5(16_5),[5,2])
 ItkConstDeclLtPatch(C220,[9,8])
 ItkConstDeclRt([9,8])
 ItkLiteral(98 StkBasedLit,(16_c8421),[9,12])
 ItkConstDeclValue([9,10])
 ItkDeclTypeAbsent([9,10])
 ItkConstDeclType([9,10])
 ItkDeclId(L9(16_9),Id6"E",L-9223372036854775808(16_8000000000000000),[9,8])
 ItkOpenDeclListSep(L4(16_4),[5,2])
 ItkConstDeclLtPatch(C163,[8,8])
 ItkConstDeclRt([8,8])
 ItkLiteral(99 StkLongBasedLit,(16_7fffffffffffffff),[8,12])
 ItkConstDeclValue([8,10])
 ItkDeclTypeAbsent([8,10])
 ItkConstDeclType([8,10])
 ItkDeclId(L9(16_9),Id5"D",L-9223372036854775808(16_8000000000000000),[8,8])
 ItkOpenDeclListSep(L3(16_3),[5,2])
 ItkConstDeclLtPatch(C111,[7,8])
 ItkConstDeclRt([7,8])
 ItkLiteral(98 StkBasedLit,(16_10ffff),[7,12])
 ItkConstDeclValue([7,10])
 ItkDeclTypeAbsent([7,10])
 ItkConstDeclType([7,10])
 ItkDeclId(L9(16_9),Id4"C",L-9223372036854775808(16_8000000000000000),[7,8])
 ItkOpenDeclListSep(L2(16_2),[5,2])
 ItkConstDeclLtPatch(C58,[6,8])
 ItkConstDeclRt([6,8])
 ItkLiteral(97 StkLongIntLit,(16_11f71fb04cb),[6,12])
 ItkConstDeclValue([6,10])
 ItkDeclTypeAbsent([6,10])
 ItkConstDeclType([6,10])
 ItkDeclId(L9(16_9),Id3"B",L-9223372036854775808(16_8000000000000000),[6,8])
 ItkOpenDeclListSep(L1(16_1),[5,2])
 ItkConstDeclLtPatch(C10,[5,8])
 ItkConstDeclRt([5,8])
 ItkLiteral(96 StkIntLit,(16_d),[5,12])
 ItkConstDeclValue([5,10])
 ItkDeclTypeAbsent([5,10])
 ItkConstDeclType([5,10])
 ItkDeclId(L9(16_9),Id2"A",L-9223372036854775808(16_8000000000000000),[5,8])
 ItkOpenScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
