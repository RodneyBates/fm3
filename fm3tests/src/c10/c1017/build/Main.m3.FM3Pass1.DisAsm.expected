 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L1(16_1),[1,0])
 ItkModuleBeginPatch(C691,L1(16_1),[19,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkLookupScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[20,2])
 ItkStmtListLt(L0(16_0),[20,2])
 ItkBlockDeclListLtPatch(C10,L14(16_e),[4,2])
 ItkBlockDeclListRt(L14(16_e),[4,2])
 ItkBlockDeclListSepPatch(C618,L13(16_d),[17,0])
 ItkConstDeclLtPatch(C618,[17,8])
 ItkConstDeclColonPatch(C624,[17,10],FALSE)
 ItkConstDeclEqualsPatch(C626,[17,10])
 ItkConstDeclRt([17,8])
 ItkWideTextLitRt(16_1),[17,12]
    (5:W"Nn\U1000ffNn":5)
 ItkWideTextLitLt(16_1),[17,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id15"N",[17,8])
 ItkBlockDeclListSepPatch(C558,L12(16_c),[16,0])
 ItkConstDeclLtPatch(C558,[16,8])
 ItkConstDeclColonPatch(C564,[16,10],FALSE)
 ItkConstDeclEqualsPatch(C566,[16,10])
 ItkConstDeclRt([16,8])
 ItkTextLitRt(16_1),[16,12]
    (6:"Mm\t Mm":6)
 ItkTextLitLt(16_1),[16,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id14"M",[16,8])
 ItkBlockDeclListSepPatch(C512,L11(16_b),[15,0])
 ItkConstDeclLtPatch(C512,[15,8])
 ItkConstDeclColonPatch(C518,[15,10],FALSE)
 ItkConstDeclEqualsPatch(C520,[15,10])
 ItkConstDeclRt([15,8])
 ItkWideCharLit(16_10ffff(W'\U10ffff'),[15,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id13"L",[15,8])
 ItkBlockDeclListSepPatch(C467,L10(16_a),[14,0])
 ItkConstDeclLtPatch(C467,[14,8])
 ItkConstDeclColonPatch(C473,[14,10],FALSE)
 ItkConstDeclEqualsPatch(C475,[14,10])
 ItkConstDeclRt([14,8])
 ItkWideCharLit(16_ffff(W'\Xffff'),[14,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id12"K",[14,8])
 ItkBlockDeclListSepPatch(C424,L9(16_9),[13,0])
 ItkConstDeclLtPatch(C424,[13,8])
 ItkConstDeclColonPatch(C430,[13,10],FALSE)
 ItkConstDeclEqualsPatch(C432,[13,10])
 ItkConstDeclRt([13,8])
 ItkCharLit(16_0('\X00'),[13,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id11"J",[13,8])
 ItkBlockDeclListSepPatch(C381,L8(16_8),[12,0])
 ItkConstDeclLtPatch(C381,[12,8])
 ItkConstDeclColonPatch(C387,[12,10],FALSE)
 ItkConstDeclEqualsPatch(C389,[12,10])
 ItkConstDeclRt([12,8])
 ItkCharLit(16_24('$'),[12,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id10"I",[12,8])
 ItkBlockDeclListSepPatch(C330,L7(16_7),[11,0])
 ItkConstDeclLtPatch(C330,[11,8])
 ItkConstDeclColonPatch(C336,[11,10],FALSE)
 ItkConstDeclEqualsPatch(C338,[11,10])
 ItkConstDeclRt([11,8])
 ItkExtendedLit(16_412ac0149999999a(876554.3x0),[11,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id9"H",[11,8])
 ItkBlockDeclListSepPatch(C279,L6(16_6),[10,0])
 ItkConstDeclLtPatch(C279,[10,8])
 ItkConstDeclColonPatch(C285,[10,10],FALSE)
 ItkConstDeclEqualsPatch(C287,[10,10])
 ItkConstDeclRt([10,8])
 ItkLongRealLit(16_405edccccccccccd(123.45d0),[10,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id8"G",[10,8])
 ItkBlockDeclListSepPatch(C232,L5(16_5),[9,0])
 ItkConstDeclLtPatch(C232,[9,8])
 ItkConstDeclColonPatch(C238,[9,10],FALSE)
 ItkConstDeclEqualsPatch(C240,[9,10])
 ItkConstDeclRt([9,8])
 ItkRealLit(16_3dcccccd(0.1),[9,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id7"F",[9,8])
 ItkBlockDeclListSepPatch(C186,L4(16_4),[8,0])
 ItkConstDeclLtPatch(C186,[8,8])
 ItkConstDeclColonPatch(C192,[8,10],FALSE)
 ItkConstDeclEqualsPatch(C194,[8,10])
 ItkConstDeclRt([8,8])
 ItkBasedLit(16_c8421(16_c8421),[8,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id6"E",[8,8])
 ItkBlockDeclListSepPatch(C134,L3(16_3),[7,0])
 ItkConstDeclLtPatch(C134,[7,8])
 ItkConstDeclColonPatch(C140,[7,10],FALSE)
 ItkConstDeclEqualsPatch(C142,[7,10])
 ItkConstDeclRt([7,8])
 ItkLongBasedLit(16_7fffffffffffffff(16_7fffffffffffffffL),[7,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id5"D",[7,8])
 ItkBlockDeclListSepPatch(C87,L2(16_2),[6,0])
 ItkConstDeclLtPatch(C87,[6,8])
 ItkConstDeclColonPatch(C93,[6,10],FALSE)
 ItkConstDeclEqualsPatch(C95,[6,10])
 ItkConstDeclRt([6,8])
 ItkBasedLit(16_10ffff(16_10ffff),[6,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id4"C",[6,8])
 ItkBlockDeclListSepPatch(C43,L1(16_1),[5,0])
 ItkConstDeclLtPatch(C43,[5,8])
 ItkConstDeclColonPatch(C49,[5,10],FALSE)
 ItkConstDeclEqualsPatch(C51,[5,10])
 ItkConstDeclRt([5,8])
 ItkLongIntLit(16_11f71fb04cb(1234567890123L),[5,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id3"B",[5,8])
 ItkConstDeclLtPatch(C10,[4,8])
 ItkConstDeclColonPatch(C16,[4,10],FALSE)
 ItkConstDeclEqualsPatch(C18,[4,10])
 ItkConstDeclRt([4,8])
 ItkIntLit(16_d(13),[4,12])
 ItkDeclTypeAbsent()
 ItkDeclId(L10(16_a),Id2"A",[4,8])
 ItkLookupScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
